source/includes/fundamentals/code-snippets/Geo.java

```diff
@@ -0,0 +1,73 @@
+package com.mycompany.app;
+
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+
+// begin exampleImports
+import java.util.Arrays;
+import org.bson.conversions.Bson;
+import com.mongodb.client.model.geojson.Point;
+import com.mongodb.client.model.geojson.Polygon;
+import com.mongodb.client.model.geojson.Position;
+import static com.mongodb.client.model.Filters.near;
+import static com.mongodb.client.model.Filters.geoWithin;
+import static com.mongodb.client.model.Projections.fields;
+import static com.mongodb.client.model.Projections.include;
+import static com.mongodb.client.model.Projections.excludeId;
+// end exampleImports
+
+public class Geo {
+
+    private final MongoClient mongoClient;
+
+    public Geo() {
+        final String uri = System.getenv("DRIVER_REF_URI");
+        mongoClient = MongoClients.create(uri);
+
+    }
+
+    public void go() {
```

Comment:
nit: not making this `public static void main` means this must be run as part of some larger application setup, a convention not agreed upon by the team.

Reply:
Thanks for the note. Fix: 

This has to do with me having a poor grasp of IntelliJ, apologies. I thought not including a main function would make this easier for readers to try this class in their study project from the quick start (I thought main function had to be in file with the same name as the project but this isn't true), but after looking into this more I see that not including a main function just adds an extra step unnecessarily.